packages/headless/src/features/commerce/facets/facet-set/facet-set-actions.ts

```diff
@@ -0,0 +1,26 @@
+import {createAction} from '@reduxjs/toolkit';
+import {
+  requiredNonEmptyString,
+  serializeSchemaValidationError,
+  validatePayloadAndThrow,
+} from '../../../../utils/validate-payload';
+import {ToggleSelectCategoryFacetValueActionCreatorPayload} from '../../../facets/category-facet-set/category-facet-set-actions';
+import {validateCategoryFacetValue} from '../../../facets/category-facet-set/category-facet-validate-payload';
+
+export const toggleSelectCommerceCategoryFacetValue = createAction(
```

Comment:
I chose to define our own action because the payload is different and it caused all sorts of issues.